Source/SVGKImage.h

```diff
@@ -111,7 +111,6 @@ typedef void (^SVGKImageAsynchronousLoadingDelegate)(SVGKImage* loadedImage);
 + (SVGKImage*) imageWithSource:(SVGKSource *)newSource; // if you have custom source's you want to use
 
 - (id)initWithContentsOfFile:(NSString *)path;
-- (id)initWithData:(NSData *)data;
```

Comment:
It's not implemented yet, but we do need this method. Please don't delete it.

If preferred, you can add an empty method body to the .m file, which NSAssert (false, @"Not implemented yet");